src/main/java/model/input/file/ExcelFile.java

```diff
@@ -72,19 +86,77 @@ private void addRows(Iterator<Row> rowIterator) {
 		}
 	}
 
-	private DataValue toDataValue(Cell cell) {
+	private DataValue toDataValue(Cell cell, ColumnInfo columnInfo) {
 		switch (cell.getCellType()) {
-			case Cell.CELL_TYPE_STRING:
-				return new StringValue(cell.getStringCellValue());
-			case Cell.CELL_TYPE_NUMERIC:
-				double cellValue = cell.getNumericCellValue();
-				return (cellValue % 1 == 0)
-						? new IntValue((int) cellValue) : new FloatValue((float) cellValue);
 			case Cell.CELL_TYPE_BLANK:
-				return new StringValue("");
-			default: throw new UnsupportedOperationException(
-					String.format("Cell type %s not supported", cell.getCellType())
-			);
+				return DataValue.getNullInstance(columnInfo.getType());
+			case Cell.CELL_TYPE_NUMERIC:
```

Comment:
Ik fix dit in de #154 reader-fix branch omdat deze change van een merge met die branch komt